assets/airports/rjbb.json

```diff
@@ -0,0 +1,2732 @@
+            ["tam", 1],
+            ["lot", 1],
+            ["dlh", 1],
+            ["mas", 1],
+            ["qfa", 3],
+            ["qtr", 1],
+            ["rja", 1],
+            ["sas", 4],
+            ["sia", 1],
+            ["swr", 1],
+            ["sht", 65],
+            ["tap", 1],
+            ["rot", 1],
+            ["tar", 1],
+            ["thy", 1],
+            ["ual", 6],
+            ["vir", 17],
+            ["vlg", 2]
+        ]
+    }, {
+        "origin": "RJBB",
+        "destination": "",
+        "category": "departure",
+        "route": "RJBB.IWAYA1.SOUJA",
+        "altitude": null,
+        "speed": null,
+        "method": "random",
+        "rate": 0.365,
+        "airlines": [
+            ["aee", 2],
+            ["ein", 4],
+            ["afl", 1],
+            ["amx/long", 1],
+            ["aca", 7],
+            ["cca", 1],
+            ["afr", 1],
+            ["aic", 4],
+            ["amc", 1],
+            ["asl", 1],
+            ["aza", 2],
+            ["ana", 1],
+            ["aal/long", 8],
+            ["aar", 1],
+            ["aua", 1],
+            ["ava/long", 1],
+            ["baw", 123],
+            ["bel", 1],
+            ["lzb", 1],
+            ["cpa", 1],
+            ["ces", 1],
+            ["csn", 1],
+            ["ctn", 3],
+            ["dal", 14],
+            ["ely", 1],
+            ["uae", 1],
+            ["etd", 1],
+            ["eva", 2],
+            ["ewg", 2],
+            ["gia", 1],
+            ["gwi", 3],
+            ["ibe", 1],
+            ["ibs", 4],
+            ["jal", 1],
+            ["jai", 2],
+            ["klm", 1],
+            ["klc", 1],
+            ["kal/long", 1],
+            ["tam", 1],
+            ["lot", 1],
+            ["dlh", 1],
+            ["mas", 1],
+            ["qfa", 3],
+            ["qtr", 1],
+            ["rja", 1],
+            ["sas", 4],
+            ["sia", 1],
+            ["swr", 1],
+            ["sht", 65],
+            ["tap", 1],
+            ["rot", 1],
+            ["tar", 1],
+            ["thy", 1],
+            ["ual", 6],
+            ["vir", 17],
+            ["vlg", 2]
+        ]
+    }, {
+        "origin": "RJBB",
+        "destination": "",
+        "category": "departure",
+        "route": "RJBB.IWAYA1.WASYU",
+        "altitude": null,
+        "speed": null,
+        "method": "random",
+        "rate": 0.365,
+        "airlines": [
+            ["aee", 2],
+            ["ein", 4],
+            ["afl", 1],
+            ["amx/long", 1],
+            ["aca", 7],
+            ["cca", 1],
+            ["afr", 1],
+            ["aic", 4],
+            ["amc", 1],
+            ["asl", 1],
+            ["aza", 2],
+            ["ana", 1],
+            ["aal/long", 8],
+            ["aar", 1],
+            ["aua", 1],
+            ["ava/long", 1],
+            ["baw", 123],
+            ["bel", 1],
+            ["lzb", 1],
+            ["cpa", 1],
+            ["ces", 1],
+    
```

Comment:
Are the `rate` values for the arrival spawnPatterns accurate? There are only 9.6 arrivals per hour, making it pretty slow. Is that expected?